htmd/smallmol/util.py

```diff
@@ -71,4 +181,227 @@ def drawIsoSurface(values3d, resolution=1., plot_center=None, viewer=None):
     outf = tempname(suffix='.cube')
     writeVoxels(values3d, outf, mincoor, maxcoor, rescoor)
     viewer.send('mol new {} type cube first 0 last -1 step 1 waitfor 1 volsets {{0 }}'.format(outf))
-    viewer.send('mol modstyle 0 top Isosurface 0.75 0 2 0 1 1')
\ No newline at end of file
+    viewer.send('mol modstyle 0 top Isosurface 0.75 0 2 0 1 1')
+
+
+def alignMol(smallmol, refmol):
+    """
+    Return a new SmallMol object aligned to a refmol that can be a htmd.smallmol.smallmol.SmallMol
+    or rdkit.Chem.rdchem.Mol. It removes all the conformers stored in the original object.
+
+
+    Parameters
+    ----------
+    smallmol: htmd.smallmol.smallmol.SmallMol
+     The SmallMol object to align
+    refmol: htmd.smallmol.smallmol.SmallMol or rdkit.Chem.rdchem.Mol
+        The molecule to align to
+
+    Return
+    ------
+    newsmallmol: htmd.smallmol.smallmol.SmallMol
+        a new SmallMol aligned to reference molecule
+    """
+
+    from htmd.smallmol.smallmol import SmallMol
+    from rdkit.Chem.rdMolAlign import GetO3A
+
+    if isinstance(refmol, SmallMol):
+        refmol = refmol.toRdkitMol(includeConformer=True)
+
+    sm_rdkit = smallmol.toRdkitMol(includeConformer=True)
+
+    pyO3A = GetO3A(sm_rdkit, refmol)
+    rmsd = pyO3A.Align()
+    print('Alignment with a RMSD of {}'.format(rmsd))
+    coords_new = sm_rdkit.GetConformer().GetPositions()
+
+    sm_new = SmallMol(smallmol, fixHs=False)
+    sm_new.removeConformers()
+    sm_new.coords = coords_new[:, :, np.newaxis]
+
+    return sm_new
+
+def inputToOutput(input_file, input_format, output_format):
```

Comment:
rename this to `openbabelConvert` or `convertFileType`. `inputToOutput` is a too generic name and makes no sense out of context

Reply:
you are completely right. Modified it